content-src/components/ContextMenu/ContextMenu.jsx

```diff
@@ -129,6 +126,8 @@ export class ContextMenuItem extends React.PureComponent {
         this.focusSibling(event.target, event.key);
         break;
       case "Enter":
+      case " ":
```

Comment:
should this be "Space" https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode

Reply:
I'll change that! I think it's in a few other places too so I'll do a quick check for them.

Reply:
So I ran into an issue - event.code causes the listener to run into issues when checking for the Shift key, which we use to determine if a `Tab` press should close the menu or not. It seems like it's more reliable to use " " and event.key, but let me know what you think!

Reply:
ah ok, thats because these are React [synthetic events](https://reactjs.org/docs/events.html#keyboard-events) that doesn't support event.code